google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditor.java

```diff
@@ -186,14 +194,22 @@ private void resetRuntimeDisplay() {
     }
   }
 
+  private void updateStagedArtifactNameEmptyText() {
+    Path artifactPath = Paths.get(Strings.nullToEmpty(deploymentSource.getFilePath()));
+    AppEngineFlexibleDeploymentArtifactType deploymentArtifactType =
+        AppEngineFlexibleDeploymentArtifactType.typeForPath(artifactPath);
+    stagedArtifactNameTextField
+        .getEmptyText()
+        .setText(deploymentArtifactType.getDefaultArtifactName());
```

Comment:
I'm being real lazy here, but are we setting the default name to the name of the deployment artifact if this is a new run config?